src/views/LoginView.vue

```diff
@@ -135,8 +202,55 @@ main {
   margin-right: 1rem;
 }
 
-label {
+.p-error {
+  display: block;
+  font-size: 12px;
+  text-align: left;
+  line-height:18px;
+}
+
+.p-error > i {
```

Comment:
Als je dan toch veel css gebruikt, doe het dan nested, maakt het wat leesbaarder imo.

Reply:
I feel like this is incorporated in the mentioning of issues #14 and #29

Reply:
Fine.